assets/template-giftcard.css

```diff
@@ -107,25 +107,25 @@ h2,
 }
 
 .button::selection {
-  background-color: rgba(var(--color-base-solid-button-labels), 0.3);
+  background-color: rgba(var(--color-button-label), 0.3);
 }
 
 .button:hover {
-  box-shadow: 0 0 0 0.2rem rgb(var(--color-base-accent-1));
+  box-shadow: 0 0 0 0.2rem rgb(var(--color-button));
 }
 
 .button--secondary {
-  color: rgb(var(--color-base-outline-button-labels));
-  box-shadow: 0 0 0 0.1rem rgb(var(--color-base-outline-button-labels));
+  color: rgb(var(--color-secondary-button-label));
+  box-shadow: 0 0 0 0.1rem rgb(var(--color-secondary-button-label));
```

Comment:
Looks good - box-shadow is being used as an outline here, not a shadow, so it's ok to use the label color.